### PR TITLE
Fix zpages

### PR DESCRIPTION
**Pull request by ramonza**

### CODE REVIEW

File: zpages/tracez.go [hunk 1]
```diff
@@ -392,3 +392,57 @@ func writeTextTraces(w io.Writer, data traceData) {
...
+}
+
+type summaryPageRow struct {
+	Name    string
+	Active  int
+	Latency []int
+	Errors  int
+}
+
+func (s *summaryPageData) Len() int           { return len(s.Rows) }
```
Comment (odeke-em) [inline zpages/tracez.go]: We use Go1.8+ and can use `sort.Slice` which is available https://github.com/golang/go/blob/cceee685be4436c8fd417e9c248429b8625b978d/api/go1.8.txt#L243

File: zpages/rpcz_test.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
+// Copyright 2017, OpenCensus Authors
```
Comment (odeke-em) [inline zpages/rpcz_test.go]: It's 2018 :)

**Comment by rakyll** [comment]
Not familiar with the summary page and adding @songy23.

**Comment by ramonza** [comment]
This summary page code is just moved between files. It was only used from tracez, not rpcz, so it makes not sense to have it in rpcz.go.

**Comment by songy23** [comment]
That's right, the open source rpcz page should only contain information on stats, not trace spans. Spans should be displayed on tracez instead.

**Pull request merged**